smdebug/core/hook.py

```diff
@@ -294,6 +294,10 @@ def _get_worker_name(self):
     def _get_num_workers(self):
         pass
 
+    @abstractmethod
+    def _is_not_supported(self):
```

Comment:
I dont see the impl

Reply:
This is an abstractmethod.
Impl here:
https://github.com/awslabs/sagemaker-debugger/blob/01c96b2ce487ecdea336387263f64a59d38f22d7/smdebug/tensorflow/session.py#L237
https://github.com/awslabs/sagemaker-debugger/blob/f1498e42f72bc4381960b350ddb0b5b46b397a5d/smdebug/tensorflow/keras.py#L61